README.md

```diff
@@ -366,7 +366,7 @@ function mapStateToProps(state) {
 }
 
 function mapDispatchToProps(dispatch) {
-  return bindActionCreators(Object.assign({}, todoActionCreators, counterActionCreators), dispatch);
+  return bindActionCreators({ ...todoActionCreators, ...counterActionCreators}, dispatch);
```

Comment:
We'd rather not use ES7 experimental code in examples to avoid confusing beginners.